### PR TITLE
Removed typehint from HasTags::convertToTags to allow Tag to be passed in

### PR DESCRIPTION
**Pull request by AlexVanderbist**

### CODE REVIEW

File: tests/HasTagsTest.php [hunk 1]
```diff
@@ -144,6 +144,20 @@ public function it_provides_as_scope_to_get_all_models_that_have_any_of_the_give
     }
 
     /** @test */
+    public function it_provides_as_scope_to_get_all_models_that_have_any_of_the_given_tag_instances()
+    {
+        $tag = Tag::findOrCreate('tagA');
```
Comment (freekmurze) [inline tests/HasTagsTest.php]: Could you use a typed tag in this test. imho that'll make it more clear that that instance of tag is used in `withAnyTags`.

File: tests/HasTagsTest.php [hunk 2]
```diff
@@ -144,6 +144,20 @@ public function it_provides_as_scope_to_get_all_models_that_have_any_of_the_give
     }
 
     /** @test */
+    public function it_provides_a_scope_to_get_all_models_that_have_any_of_the_given_tag_instances()
+    {
+        $tag = Tag::findOrCreate('tagA', 'typeA');
```
Comment (AlexVanderbist) [inline tests/HasTagsTest.php]: @freekmurze like this?

Comment (freekmurze) [inline tests/HasTagsTest.php]: Yup thanks!

**Comment by freekmurze** [comment]
👍

**Pull request merged**